samples/application/simple-go/main.go

```diff
@@ -200,7 +200,19 @@ func main() {
 
 	// Invoke FPC Chaincode
 	logger.Infof("--> Invoke FPC chaincode: %s", contract.Name())
-	result, err := contract.SubmitTransaction("myFunction", "arg1", "arg2", "arg3")
+	var funcion, arg1, arg2, arg3 string
+	if contract.Name() == "echo" {
+		funcion = "myFunction"
+		arg1 = "arg1"
+		arg2 = "arg2"
+		arg3 = "arg3"
+	} else {
+		funcion = "storeAsset"
+		arg1 = "asset1"
+		arg2 = "200"
+		arg3 = ""
+	}
```

Comment:
this change looks a bit ugly to me. What is someone uses the test-network to deploy echo chaincode with a different name or has even multiple echo chaincodes deployed, eg., echo1 and echo2 .... this is not really robust I believe. 

Instead of touching simple-go, why don't we explain in helloworld how to modify simple-go (i.e., change the args to work with helloworld). Also [simple-cli](https://github.com/hyperledger/fabric-private-chaincode/tree/main/samples/deployment/test-network#how-to-use-simple-cli-go) could be used to demonstrate helloworld chaincode with test network.

Reply:
This will be covered in #605